changelog/6387.feature.rst

```diff
@@ -0,0 +1 @@
+Enable the usage of ``super().from_parent`` for ``Node`` subclasses.
```

Comment:
Since `from_parent` has not been released yet this could go with the previous changelog, and/or is not necessary / worth mentioning in the first place?  (this might have been mentioned already with some outdated code IIRC).

Reply:
applied, thanks